package.json

```diff
@@ -36,18 +36,9 @@
   "dependencies": {
     "eslint-config-metarhia": "^6.0.0",
```

Comment:
Also, why this is not in devDependencies like in [common](https://github.com/metarhia/common/blob/master/package.json#L40)?